### PR TITLE
fix(UserMenu): Fix address display issue

### PR DESCRIPTION
**Pull request by Woody810**

### CODE REVIEW

File: packages/uikit/src/widgets/Menu/components/UserMenu/index.tsx [hunk 1]
```diff
@@ -79,11 +79,11 @@ const UserMenu: React.FC<UserMenuProps> = ({
...
   const [tooltipRef, setTooltipRef] = useState<HTMLDivElement | null>(null);
-  const accountEllipsis = account
-    ? isSidName
-      ? account
-      : `${account.substring(0, 2)}...${account.substring(account.length - 4)}`
-    : null;
+
+  const isNeedEllipsis = account && !isSidName;
+  const accountEllipsis = account ? `${account.substring(0, 2)}...${account.substring(account.length - 4)}` : null;
+  const accountSID = account && isSidName ? account : null;
```
Comment (chef-jojo) [inline packages/uikit/src/widgets/Menu/components/UserMenu/index.tsx:85]: uikit doesn't need to know isSidName
it also let network switcher not having text

File: apps/web/src/components/Menu/UserMenu/index.tsx [hunk 2]
```diff
@@ -95,7 +95,13 @@ const UserMenu = () => {
 
   if (account) {
     return (
-      <UIKitUserMenu account={sidName || account} avatarSrc={avatarSrc} text={userMenuText} variant={userMenuVariable}>
+      <UIKitUserMenu
+        account={account}
+        sidName={sidName}
```
Comment (chef-jojo) [inline apps/web/src/components/Menu/UserMenu/index.tsx:100]: we should remove the `sidName` from uikit
the uikit menu can add a `ellipsis` flag when sidName

Comment (chef-jojo) [reply apps/web/src/components/Menu/UserMenu/index.tsx:100]: `<UIKitUserMenu  account={sidName || account} ellipsis={!!sidName}  />`

Comment (Woody810) [reply apps/web/src/components/Menu/UserMenu/index.tsx]: done

Comment (chef-jojo) [reply apps/web/src/components/Menu/UserMenu/index.tsx]: ellipsis can be default true, or else bridge will show full address

Comment (Woody810) [reply apps/web/src/components/Menu/UserMenu/index.tsx]: done

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**